packages/contracts-bedrock/test/L1/OPContractsManager.t.sol

```diff
@@ -2083,3 +2101,134 @@ contract OPContractsManager_SetRC_Test is OPContractsManager_Upgrade_Harness {
         opcm.setRC(true);
     }
 }
+
+/// @title OPContractsManager_L1ContractsRelease_Test
+/// @notice Tests the `l1ContractsRelease` function of the `OPContractsManager` contract.
+contract OPContractsManager_L1ContractsRelease_Test is OPContractsManager_TestInit {
+    /// @notice Tests that l1ContractsRelease returns the correct value when isRC is true.
+    function test_l1ContractsRelease_rcTrue_succeeds() public view {
+        // isRC is true by default
+        string memory release = opcm.l1ContractsRelease();
+        bytes memory releaseBytes = bytes(release);
+        assertEq(Bytes.slice(releaseBytes, releaseBytes.length - 3, 3), "-rc", "release should end with '-rc'");
+    }
+}
+
+/// @title OPContractsManager_Validate_Test
+/// @notice Tests the `validate` function of the `OPContractsManager` contract.
+contract OPContractsManager_Validate_Test is OPContractsManager_TestInit {
+    /// @notice Tests that validate function returns validation results.
+    function test_validate_succeeds() public view {
```

Comment:
I think this is generally ok, although since this is basically just a wrapper for the StandardValidator it means we potentially risk duplicating a ton of tests. I think it's okay.